### ISSUE TITLE
Login Warning

### ISSUE THREAD

**Issue by abcajax** [WP-API/Basic-Auth#68]
I have implemented this plugin. It works great for non logged in user but for logged in user used via Postman, it shows the following. Could you please help solve this. I have latest wordpress install till date.

Login Warning
There was an issue with your log in. Your user account has logged in recently from a different location.

**Comment by cuxaro** [comment]
> I have implemented this plugin. It works great for non logged in user but for logged in user used via Postman, it shows the following. Could you please help solve this. I have latest wordpress install till date.
> 
> 
> 
> Login Warning
> 
> There was an issue with your log in. Your user account has logged in recently from a different location.
> 
> 

Hello, have you this code in your .htaccess?


RewriteEngine on
RewriteCond %{HTTP:Authorization} ^(.*)
RewriteRule ^(.*) - [E=HTTP_AUTHORIZATION:%1]

**Comment by abcajax** [comment]
Earlier, I added RewriteRule .* - [E=REMOTE_USER:%{HTTP:Authorization}] but I commented and tried your suggested ones but still the same problem.

**Comment by abcajax** [comment]
It only happens to the normal users trying to access via REST. It works fine for the admin user. So some kind of issue with the normal user.

**Comment by cuxaro** [comment]
Are you sure that haven't any other plugin that limit the number of simultaneous users?

**Comment by abcajax** [comment]
no, I tried disabling all the possible plugins which can cause issue and tried. Just left the standard plugins but still the problem. I had wordfence installed but I disabled it.

**Comment by Keeyan** [comment]
@abcajax @cuxaro any luck? Having the same issue and tried the Rewrite Rules.
